docs/content/concepts/assets/asset-checks/checking-for-data-freshness.mdx

```diff
@@ -45,15 +47,15 @@ Before continuing, you should be familiar with:
 
 ---
 
-## Defining freshness checks for source assets
+## Defining freshness checks for assets kept up-to-date outside of Dagster
```

Comment:
This is quite a mouthful -- my initial reaction was to replace this with "external assets", but right now external assets are explicitly those which have no materialize or observe function. I think we need to think a bit more about the overall taxonomy here.

IMO, in a world where Dagster has a lot of assets which fall into the bucket of "have no materialization function, but might be possible to observe", it seems useful to have a way to label that category such that you have "software-defined assets" and "<X> assets". IMO, "external asset" feels like a reasonable term for any asset which cannot be materialized from within Dagster. It is in some way managed by an external system, regardless of if we can observe that system or not.

So if we're categorizing things I'd argue we could have a top level dichotomy of "materializable or external", and then "observable" is an independent property on top of that core dichotomy. You could of course have an external asset which is possible to observe, but it'd also be possible to make an observation function for a materializable asset, thus making it observable as well as materializable (I'd imagine this would be significantly rarer).

Reply:
I wrote these words a while ago and, in addition to being a mouthful, your comment made me notice that I think they're not actually accurate either: they don't account for the case where an asset is kept-up-to-date outside of Dagster, but materializations are reported by the external system that keeps them up to date.

Going to take a stab at rewording this.